Source/WebCore/platform/graphics/FontPlatformData.cpp

```diff
@@ -114,4 +115,68 @@ Vector<FontPlatformData::FontVariationAxis> FontPlatformData::variationAxes(Shou
 }
 #endif
 
+#if USE(CORE_TEXT)
```

Comment:
Might it be better to move this into its `Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp`?